apps/web/src/lang/whitelist.yml

```diff
@@ -32,3 +32,4 @@
 - kvpRgN # 'Classifications' - Title for classifications
 - rwNxlI # 'Notes' - Notes label
 - EK+25s # 'Notes' - Notes in adminMessages constant
+- caSdGd # 'Notes' - Pool Candidate table column
```

Comment:
Please clean-up the whitelist too to match what translation ids exist

Reply:
Oops, forgot about that 🤦‍♀️ Removed in [remove unused id's in whitelist](https://github.com/GCTC-NTGC/gc-digital-talent/pull/7215/commits/5782e7e06d08a27e38c23c55d825f00801b16a7b)